src/frontend/src/components/SEO.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet';
+import { useStaticQuery, graphql } from 'gatsby';
+
+function SEO({ description, lang, meta, title }) {
+  const { site } = useStaticQuery(
+    graphql`
+      query {
+        site {
+          siteMetadata {
+            title
+            description
+            author
+          }
+        }
+      }
+    `
+  );
+
+  const metaDescription = description || site.siteMetadata.description;
```

Comment:
Don't we already have this query from an existing [hook](https://github.com/Seneca-CDOT/telescope/blob/78610d4538200146bb5bd51b1cd8281d0f35a06b/src/frontend/src/hooks/use-site-metadata.js#L9)?

Reply:
I think the only we need from this is `title`, not `description` and `author`